app/models/dossier.rb

```diff
@@ -640,7 +644,12 @@ def expiration_notification_date
 
   def close_to_expiration?
     return false if en_instruction?
-    expiration_notification_date < Time.zone.now
+    expiration_notification_date < Time.zone.now && expiration_notification_date > Expired::REMAINING_WEEKS_BEFORE_EXPIRATION.weeks.ago
```

Comment:
proposition pour conserver la comparaison `<` plus facile a lire avec le plus petit a gauche

```suggestion
    expiration_notification_date < Time.zone.now &&  Expired::REMAINING_WEEKS_BEFORE_EXPIRATION.weeks.ago < expiration_notification_date
```